docs/guides/nrfconnect_examples_software_update.md

```diff
@@ -3,13 +3,73 @@
 Some examples for the development kits from Nordic Semiconductor support
 over-the-air Device Firmware Upgrade.
 
-Currently, the Bluetooth LE is the only available transport for performing the
-DFU operation and it uses nRF Connect SDK's
-[Simple Management Protocol](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/device_mgmt/index.html#device-mgmt).
-The upgrade can be done either using a smartphone application or a PC command
-line tool.
+Depending on the example, the DFU can be performed using the Matter-compliant
+OTA feature in which the Matter operational network is used for downloading a
+new firmware image, or using
+[Simple Management Protocol](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/device_mgmt/index.html#device-mgmt)
+over Bluetooth LE. In the latter case, the DFU can be done either using a
+smartphone application or a PC command line tool.
 
-## Device Firmware Upgrade using smartphone
+## Devide Firmware Upgrade over Matter
+
+> **_NOTE:_** The procedure presented below requires that you have OpenThread
+> Border Router set up either in Docker or on a Raspberry Pi. Read
+> [Setup OpenThread Border Router on Raspberry Pi](openthread_border_router_pi.md)
+> to learn how to install the OTBR on a Raspberry Pi.
+
+To test the DFU over Matter, complete the following steps:
+
+1.  Navigate to the CHIP root directory.
+2.  Build OTA Provider application for Linux:
```

Comment:
What is lacking for me in the whole section is a little bit of explanation how the topology looks like, so saying that OTA provider is separate node from the controller (chip-tool) and Matter accessory is OTA Requestor.

Reply:
Added some explanation.